kie-dmn-core/src/test/java/org/kie/dmn/core/DMNRuntimeTest.java

```diff
@@ -733,7 +733,7 @@ public void testInvalidVariableNames() {
         DMNModel dmnModel = runtime.getModel( "https://github.com/droolsjbpm/kie-dmn", "0001-input-data-string" );
         assertThat( dmnModel, notNullValue() );
         assertThat( dmnModel.hasErrors(), is(true) );
-        assertThat( formatMessages( dmnModel.getMessages() ), dmnModel.getMessages().size(), is(4) );
+        assertThat( formatMessages( dmnModel.getMessages() ), dmnModel.getMessages().size(), is(10) );
```

Comment:
Yes, I didn't want to each each message string individually because the strings will change, specially with i18n. Lets discuss today the consolidation of the messages and although I am not a big fan, I think we will need to create an identifier code for error messages. :(